moveit_simple/include/moveit_simple/moveit_simple.h

```diff
@@ -87,6 +87,17 @@ class Robot
 public:
    Robot(const ros::NodeHandle & nh, const std::string &robot_description,
                                      const std::string &group_name);
+  
+  /**
+  * @brief Constructor for the case where the IK implementation does not match the
+  * SRDF. For example: IKFast solutions are generally solved for the base_link to
+  * tool0 of a robot, and the robot_description is defined for some world_frame to
+  * some tcp frame.
+  */   
```

Comment:
Add a note that the transforms to the IK frames are assumed to be static.  It would be nice if TF told us if a particular transform was static.  Then the class could automatically adapt to static/non-static transforms.

Reply:
Done.